source/_integrations/xiaomi_tv.markdown

```diff
@@ -51,5 +56,6 @@ media_player:
 ```
 
 <div class='note info'>
-The platform will never turn your TV off. Instead, it will be put to sleep and woken up. This can be useful, because the selected source of the TV will remain the same. It will essentially turn your TV into a dumb TV.
+If the `assume_state` (default) is True the platform will never turn your TV off. Instead, it will be put to sleep and woken up. This can be useful, because the selected source of the TV will remain the same. It will essentially turn your TV into a dumb TV.
```

Comment:
```suggestion
If the `assume_state` (default) is `true` the platform will never turn your TV off. Instead, it will be put to sleep and woken up. This can be useful because the selected source of the TV will remain the same. It will essentially turn your TV into a dumb TV.
```